tests/test_catalyst.py

```diff
@@ -0,0 +1,151 @@
+import unittest
+import collections
+import numpy as np
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision
+import torchvision.transforms as transforms
+
```

Comment:
You need to add `import catalyst` here to be able to call `catalyst.__version__`